### ISSUE TITLE
Dagstuhl ChoirSet

### ISSUE THREAD

**Issue by sebastianrosenzweig** [mir-dataset-loaders/mirdata#465]
@helenacuesta and I would like to add [Dagstuhl ChoirSet](http://doi.org/10.5334/tismir.48) to mirdata.

The dataset is publicly available on [Zenodo](https://doi.org/10.5281/zenodo.3956666).

**Comment by helenacuesta** [comment]
After our discussion earlier today, we would suggest the following index structure:

```
{
"version": 1.1,
  "tracks": {
     "DCS_LI_FullChoir_Take01_A2": {
          "audio_dyn": ("audio_wav_22050_mono/DCS_LI_FullChoir_Take01_A2_DYN.wav", checksum),
          "audio_hsm": ("audio_wav_22050_mono/DCS_LI_FullChoir_Take01_A2_HSM.wav", checksum),
          "audio_lrx": ("audio_wav_22050_mono/DCS_LI_FullChoir_Take01_A2_LRX.wav", checksum),
          "f0_crepe_dyn": ("annotations_csv_F0_CREPE/DCS_LI_FullChoir_Take01_A2_DYN.csv", checksum),
          "f0_crepe_hsm": ("annotations_csv_F0_CREPE/DCS_LI_FullChoir_Take01_A2_HSM.csv", checksum),
          "f0_crepe_lrx": ("annotations_csv_F0_CREPE/DCS_LI_FullChoir_Take01_A2_LRX.csv", checksum),
          "f0_pyin_dyn": ("annotations_csv_F0_PYIN/DCS_LI_FullChoir_Take01_A2_DYN.csv", checksum),
          "f0_pyin_hsm": ("annotations_csv_F0_PYIN/DCS_LI_FullChoir_Take01_A2_HSM.csv", checksum),
          "f0_pyin_lrx": ("annotations_csv_F0_PYIN/DCS_LI_FullChoir_Take01_A2_LRX.csv", checksum),
          "score": ("annotations_csv_scorerepresentation/DCS_LI_FullChoir_Take01_Stereo_STM_S.csv", checksum) # this points to the same score representation for both A1 and A2
     },

     "DCS_LI_FullChoir_Take01_A1": {
     	  "audio_lrx": ("audio_wav_22050_mono/DCS_LI_FullChoir_Take01_A1_LRX.wav", checksum),
     	  "f0_crepe_lrx": ("annotations_csv_F0_CREPE/DCS_LI_FullChoir_Take01_A1_LRX.csv", checksum),
     	  "f0_pyin_lrx": ("annotations_csv_F0_PYIN/DCS_LI_FullChoir_Take01_A1_LRX.csv", checksum),
     	  "score": ("annotations_csv_scorerepresentation/DCS_LI_FullChoir_Take01_Stereo_STM_S.csv", checksum)
     },

     "DCS_LI_FullChoir_Take01_S1" : {...},
     "DCS_LI_FullChoir_Take01_S2" : {...},
     "DCS_LI_FullChoir_Take01_T1" : {...},
     "DCS_LI_FullChoir_Take01_T2" : {...},
     "DCS_LI_FullChoir_Take01_B1" : {...},
     "DCS_LI_FullChoir_Take01_B2" : {...},
  },

  "multitracks": {
    "DCS_LI_FullChoir_Take01": {
         "tracks": ["DCS_LI_FullChoir_Take01_A2",
                    "DCS_LI_FullChoir_Take01_A1",
                    "DCS_LI_FullChoir_Take01_S1",
                    "DCS_LI_FullChoir_Take01_S2",
                    "DCS_LI_FullChoir_Take01_T1",
                    "DCS_LI_FullChoir_Take01_T2",
                    "DCS_LI_FullChoir_Take01_B1",
                    "DCS_LI_FullChoir_Take01_B2"],

         "audio_stm": ("audio_wav_22050_mono/DCS_LI_FullChoir_Take01_Stereo_STM.wav", checksum),
         "audio_str": ("audio_wav_22050_mono/DCS_LI_FullChoir_Take01_Stereo_STR.wav", checksum),
         "audio_stl": ("audio_wav_22050_mono/DCS_LI_FullChoir_Take01_Stereo_STL.wav", checksum),
         "audio_rev": ("audio_wav_22050_mono/DCS_LI_FullChoir_Take01_StereoReverb_STM.wav", checksum),
         "beat": ("annotations_csv_beat/DCS_LI_FullChoir_Take01_Stereo_STM.csv", checksum),
     },
    "DCS_LI_FullChoir_Take02": ...
  },
}

```
Please, let us know whether this mock-up structure makes sense (it's only one multitrack).

**Comment by rabitt** [comment]
@helenacuesta @sebastianrosenzweig the index structure looks great to me!

**Comment by sebastianrosenzweig** [comment]
Please see Pull Request #481

**Comment by rabitt** [comment]
Closed by #481
